indico/web/client/js/index.js

```diff
@@ -12,3 +12,8 @@ import './legacy/indico.js';
 import './legacy/timetable.js';
 
 import '../styles/screen.scss';
+
+import React from 'react';
```

Comment:
i don't think this is needed. we'll import it wherever we actually want to use it.